modules/ROOT/pages/managing-api-clients.adoc

```diff
@@ -3,16 +3,36 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic provides an overview of client management on Anypoint Platform. As an API owner, you can apply an OAuth 2.0 policy to authenticate client applications that try to access your API. You need an OAuth provider to use an OAuth 2.0 policy. One of the following OAuth providers is recommended:
+This topic provides an overview of client management on Anypoint Platform. As an API owner, you can apply an OAuth 2.0 policy to authenticate client applications that try to access your API. You need an OAuth provider to use an OAuth 2.0 policy.
 
-* OpenAM version: 14
-* PingFederate versions: 6 - 8
-* Dynamic Client Registration-compliant identity providers
+Using the client management, you can configure several client providers to manage client credentials.
+Any client providers you configure are available in your organization.
```

Comment:
Same as above. Unless you say something like "immediately available" or give more color, this doesn't add much information.

Reply:
```suggestion
```